### ISSUE TITLE
Purchase ticket: Tickets overview section showing wrong ticket count

### ISSUE THREAD

**Issue by amassarwi** [decred/decrediton#2525]
Currently when purchasing a new ticket I see that I have *0* live tickets, which isn't correct:

![image](https://user-images.githubusercontent.com/10324528/84378800-09e3b580-abe5-11ea-9965-5b497b4413dc.png)

**Comment by guilhermemntt** [comment]
Right, is SPV enabled? If so, maybe that's the case https://github.com/decred/decrediton/issues/1959.

**Comment by vctt94** [comment]
Maybe related to #2520

**Comment by amassarwi** [comment]
I have many live tickets showing on the tickets page , on the other hand the count is showing 0, so it looks like it's not realted to the two issues you guys mentioned.

**Comment by vctt94** [comment]
So the bug does not happen right after buying a ticket? For me it shows the properly number of tickets

**Comment by amassarwi** [comment]
No, it's just showing wrong number in my case.

**Comment by vctt94** [comment]
And it is broken at this view? 

![image](https://user-images.githubusercontent.com/15069783/84389291-34754480-abcc-11ea-83a7-066b1557b525.png)

**Comment by amassarwi** [comment]
i refered to this view, this is where my screenshot is taken from as well, as i said all tickets list displaying tickets  properly but not the section in the purchase ticket view

**Comment by sumiflow** [comment]
I think I'm having the same issue running 1.6.3 in SPV mode.  Rescanning and closing and re-opening Decrediton do not fix the issue.  See screenshots below.
![image](https://user-images.githubusercontent.com/5420838/120902936-babe3e00-c608-11eb-83b2-e227877fd6d8.png)
